### ISSUE TITLE
[YSQL] FATALs on Toggling YSQL packed columns on and off every 30 mins while continuing workloads throughout

### ISSUE THREAD

**Issue by kripasreenivasan** [yugabyte/yugabyte-db#14310]
Jira Link: [DB-3747](https://yugabyte.atlassian.net/browse/DB-3747)
### Description

Steps:

1. Created a cluster with packed columns disabled and nodes in three regions- AP, EU, US
2. Started workloads encompassing many datatypes, sequences,  DMLs
3. Edited the gflags to enable packed columns while workloads continued
4. Continued workloads after packed columns was enabled
5. After a while disabled packed columns, continued workloads all along
6. Enabled packed columns
7. Executed flush and manual compaction
8. Continued workloads
Observations:
Ran into FATAL
```
F20221004 10:25:26 ../../src/yb/yql/pggate/pggate.cc:1695] Check failed: _s.ok() Bad status: Aborted (yb/rpc/reactor.cc:94): Shutdown connection (system error 108)
    @     0x7f8c4b2fcfdb  google::LogMessage::SendToLog()
    @     0x7f8c4b2fe0a8  google::LogMessage::Flush()
    @     0x7f8c4b2fe51f  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f8c50238586  yb::pggate::PgApiImpl::ClearSeparateDdlTxnMode()
    @           0xb7499f  YBTxnDdlProcessUtility
    @           0x9ada94  PortalRunUtility
    @           0x9ad3e8  PortalRunMulti
    @           0x9acafd  PortalRun
    @           0x9a8416  yb_exec_simple_query_impl
    @           0x9a8b78  yb_exec_query_wrapper_one_attempt
    @           0x9a374b  PostgresMain
    @           0x8ef2db  BackendRun
    @           0x8ee772  ServerLoop
    @           0x8ea7ea  PostmasterMain
    @           0x7f7631  PostgresServerProcessMain
    @           0x506b82  main
    @     0x7f8c4f51b825  __libc_start_main
    @           0x506a99  _start
```

CC: @spolitov , @shamanthchandra-yb

**Comment by rthallamko3** [comment]
From Kripa
```
Adding more context to the colocated table creation issue I ran into…
I fired a script to create 5000 tables in a db. It started hitting this FATAL and one node went down after 3140 tables were created-
F20221004 11:58:03 ../../src/yb/yql/pggate/pggate.cc:1695] Check failed: _s.ok() Bad status: Network error (yb/rpc/connection.cc:196): Connect timeout Connection (0x000000000302fe78) client 172.151.54.172:36662 => 172.151.54.172:9100, passed: 14.995s, timeout: 15.000s: kConnectFailed (network error 1)
    @     0x7f02f0451fdb  google::LogMessage::SendToLog()
    @     0x7f02f04530a8  google::LogMessage::Flush()
    @     0x7f02f045351f  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f02f538d586  yb::pggate::PgApiImpl::ClearSeparateDdlTxnMode()
    @           0xb7499f  YBTxnDdlProcessUtility
    @           0x7cb6f0  _SPI_execute_plan
    @           0x7cb15b  SPI_execute
    @     0x7f02e217d734  exec_stmt
    @     0x7f02e217dbce  exec_stmt
    @     0x7f02e217a7dc  exec_stmt_block
    @     0x7f02e2179237  plpgsql_exec_function
    @     0x7f02e2174522  plpgsql_inline_handler
    @           0xb4bf42  OidFunctionCall1Coll
    @           0x9af3b8  standard_ProcessUtility
    @           0x9b2d7e  YBProcessUtilityDefaultHook
    @     0x7f02efc6c000  pgss_ProcessUtility
    @     0x7f02efc62d27  ybpgm_ProcessUtility
    @     0x7f02efc467ee  pg_hint_plan_ProcessUtility
    @           0xb7490f  YBTxnDdlProcessUtility
    @           0x9ada94  PortalRunUtility
    @           0x9ad3e8  PortalRunMulti
    @           0x9acafd  PortalRun
    @           0x9a8416  yb_exec_simple_query_impl
    @           0x9a8b78  yb_exec_query_wrapper_one_attempt
    @           0x9a374b  PostgresMain
    @           0x8ef2db  BackendRun
    @           0x8ee772  ServerLoop
    @           0x8ea7ea  PostmasterMain
    @           0x7f7631  PostgresServerProcessMain
    @           0x506b82  main
    @     0x7f02f4670825  __libc_start_main
    @           0x506a99  _start
 I’m now facing this error and I’m not able to connect to the DB though all the nodes are back up now.
yugabyte=# \c  db_001
SSL SYSCALL error: EOF detected
Previous connection kept
```

**Comment by bmatican** [comment]
@kripasreenivasan can we rename this, to not mention packed columns, now that we know it's a more generic issue?
And can you confirm if the same script, without packed, hits the same issue?

**Comment by fizaaluthra** [comment]
Closing as part of #13128  -- addressed in commit 5a6d54a6995b3d45dbba257abde296e91fa63e54
